spec/controllers/api/v1/coffeeshops_form_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.describe Api::V1::CoffeeshopsController, type: :controller do
```

Comment:
This file should be `coffeeshops_controller_spec` as it would contain tests for all actions in this API controller, it's not testing the form

Reply:
file renamed to `coffeeshops_controller_spec`